src/plugins/vis_type_xy/public/utils/use_color_picker.tsx

```diff
@@ -61,18 +64,31 @@ export const useColorPicker = (
         onClose();
       };
 
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      const handleOutsideClick = useCallback(
```

Comment:
Are you sure that you need any of this logic that checks the DOM directly? I tried locally to remove it and the behavior still worked.

Reply:
Yup you're right. I was attempting to make it work with `EuiOutsideClickDetector` inside of `EuiWrappingPopover` at first and needed the ref to check but not any more. Thanks see 69e7769.